static/scripts/dashboard/dashboardCtrl.js

```diff
@@ -848,9 +848,24 @@
                   systemIntegration.data
                 );
                 $scope.vm.installForm[sysIntName][masterName].isEnabled = true;
-                if (sysIntName === 'auth')
-                  $scope.vm.installForm[sysIntName][masterName].callbackUrl =
-                    systemIntegration.data.wwwUrl + '/' + sysIntName +'/' + systemIntegration.id + '/identify';
+                if (sysIntName === 'auth') {
+                  if (masterName === 'bitbucketKeys') {
```

Comment:
I think this will be cleaner if we build an object with `masterName` as the key.